### DISCUSSION TITLE
loading indicators are always showing

### DISCUSSION THREAD

**Discussion by tomschlick** [filamentphp/filament#9853] [Help]
So after upgrading from version 2 to 3 of the tables component, I've been seeing the loading indicator on every table's search box and each rows actions loading by default.

![image](https://github.com/filamentphp/filament/assets/70184/e9996e3d-e86d-4ef7-b2f1-10d6f503b353)

If I click the items it removed the icon but keeps the spinning indicator, then reverts back to the default once the request is complete.

I haven't been able to find anything about this via issue/discussion search or via Google. I'm thinking it has to be some kind of config or asset build issue but I haven't been able to track it down.

**Comment by leandrocfe** [comment]
What is the Filament version on the Dashboard page?

**Reply by tomschlick** [reply]
```
composer show | grep filament
filament/actions                                    v3.0.101  Easily add beautiful action modals to any Livewire component.
filament/filament                                   v3.0.101  A collection of full-stack components for accelerated Laravel app development.
filament/forms                                      v3.0.101  Easily add beautiful forms to any Livewire component.
filament/infolists                                  v3.0.101  Easily add beautiful read-only infolists to any Livewire component.
filament/notifications                              v3.0.101  Easily add beautiful notifications to any Livewire app.
filament/support                                    v3.0.101  Core helper methods and foundation code for all Filament packages.
filament/tables                                     v3.0.101  Easily add beautiful tables to any Livewire component.
filament/upgrade                                    v3.0.101  Upgrade Filament v2 code to Filament v3.
filament/widgets                                    v3.0.101  Easily add beautiful dashboard widgets to any Livewire component.
```

livewire version `v3.2.1`    -- laravel version `v10.33.0`

Also, this is not via the admin panel builder, but by using the tables components directly.

**Reply by leandrocfe** [reply]
Did you publish some Filament view?

**Reply by hmihail** [reply]
DId you find a solution for this? I have the same issue

**Reply by tomschlick** [reply]
I was not able to figure it out. Instead i published the spinner filament view and commented the spinner out so it wouldnt display anymore. It sucks because that means the spinner doesnt show for real actions, but its better for now while I look into this further.

**Reply by leandrocfe** [reply]
Sorry, what is your Filament version now?

**Reply by hmihail** [reply]
```
filament/actions                            v3.0.80  Easily add beautiful action modals to any Livewire component.
filament/forms                              v3.0.80  Easily add beautiful forms to any Livewire component.
filament/infolists                          v3.0.80  Easily add beautiful read-only infolists to any Livewire component.
filament/notifications                      v3.0.80  Easily add beautiful notifications to any Livewire app.
filament/support                            v3.0.80  Core helper methods and foundation code for all Filament packages.
filament/tables                             v3.0.80  Easily add beautiful tables to any Livewire component.
```

**Reply by leandrocfe** [reply]
You should add `^` in the composer.json
```php
"filament/filament": "^3.1"
```
and run `composer update` again
If you have filament/forms in the composer.json, remove it as well

**Reply by hmihail** [reply]
I did that, but nothing changes. Spinners are still there.

**Reply by leandrocfe** [reply]
share the composer.json

**Reply by hmihail** [reply]
[composer.json](https://github.com/filamentphp/filament/files/13874303/composer.json)

Here it is.

**Reply by leandrocfe** [reply]
Are you using only the table builder?

**Reply by hmihail** [reply]
No, I am using the forms and notifications also.

**Reply by leandrocfe** [reply]
they aren't in the composer.json

**Reply by hmihail** [reply]
No, but they work. Should I add them in composer?

**Reply by leandrocfe** [reply]
I think it isn't required if you are using forms in the table actions, because the table uses `InteractsWithForms`
Did you follow this section? https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component#adding-the-table

Open a new help topic please and share the whole code you are trying.

**Reply by hmihail** [reply]
I am also using Forms in a separate component directly. I have set up the component and the form, following the steps you mentioned.

**Reply by axkuechler** [reply]
@tomschlick Is there a solution for this in the meantime?

I'm using the actions package directly in a custom component and facing the same issue as described in the thread.

**Reply by rodrigojusto** [reply]
I has the same problem, I solved changing in config\filament.php
'livewire_loading_delay' => 'none',

**Reply by mikerockett** [reply]
Got the same issue here, only its on a frontend view using Filament actions. I have styles and scripts injected. Clicking on the action works. Just this loading indicator that won't go away.

**Reply by Bensji** [reply]
I fixed it by adding

`.fi-btn-icon.transition, .fi-icon-btn-icon.animate-spin {
	@apply hidden;
}`

To my styles myself.

**Reply by mikerockett** [reply]
It also shows on the button in the form of my action modal. I shouldn't need to hack it like this.

Also, `livewire_loading_delay` doesn't work.

**Reply by mikerockett** [reply]
This solved it: https://github.com/filamentphp/filament/discussions/7722#discussioncomment-8908970

**Reply by hmihail** [reply]
I am on livewire 3.4.9 and the spinners are still showing for me.

**Reply by leandrocfe** [reply]
It appears there might be a setup issue. I recommend trying this in a fresh Laravel/ TableBuilder installation to see if it resolves the problem.

**Reply by hmihail** [reply]
I did not had `@livewireStyles` directive at all in my layout. I added that right after `@filamentStyles` and now it's working. I apologise for wasting your time!
